meta-rz-bsp/recipes-bsp/trusted-firmware-a/trusted-firmware-a-renesas.inc

```diff
@@ -5,6 +5,14 @@ PROVIDES = "virtual/trusted-firmware-a"
 
 EXTRA_OEMAKE:append = "${TFA_EXTRA_OEMAKE}"
 
+# In recipes-bsp/trusted-firmware-a/trusted-firmware-a.inc in meta-arm, the
+# LDOPTS variable in the fiptool Makefile is modified to pass the correct
+# arguments to the linker. However, we're building v2.9.0 where the variable
+# LDLIBS is used instead, so we need to modify this ourselves.
+do_compile:prepend() {
+	sed -i '/^LDLIBS/ s,$, \$\{BUILD_LDFLAGS},' ${S}/tools/fiptool/Makefile
+}
+
```

Comment:
Hi @pbrkr 

Thanks a lot for your PR.
Going from your comments I'm guessing that the code is okay as-is for TF-A v2.10+?
Should we move this snippet to trusted-firmware-a-renesas_2.9.0.bb?

Chris